lib/pay/paddle/webhooks/subscription_payment_succeeded.rb

```diff
@@ -0,0 +1,41 @@
+module Pay
+  module Paddle
+    module Webhooks
+      class SubscriptionPaymentSucceeded
+
+        def initialize(data)
+          billable = Pay.find_billable(processor: :paddle, processor_id: data["user_id"])
+          return unless billable.present?
+          return if billable.charges.where(processor_id: data["subscription_payment_id"]).any?
+
+          charge = create_charge(billable, data)
+          notify_user(billable, charge)
+          charge
+        end
+
+        def create_charge(user, data)
+          charge = user.charges.find_or_initialize_by(
+            processor: :paddle,
+            processor_id: data["subscription_payment_id"]
+          )
+
+          charge.update(
+            amount: Integer(data["sale_gross"].to_f * 100),
+            card_type: data["payment_method"],
```

Comment:
`payment_method` will be one of `card, paypal, free, apple-pay, wire-transfer`, not sure if it fits in this field

Reply:
Yes, storing the payment method in a card type column is definitely misleading. I'll second that. But this is consistent to the existing Braintree integration which stores "PayPal" in the card_type column as well.

Maybe it would be better to have more general columns for payment method and payment method meta data.  But I didn't want to alter too much with this PR.

Reply:
Yeah, we need to improve this in Pay at some point. Braintree stores "paypal" in `card_type` and the paypal email in `card_last4` which isn't great.